dev-util/jenkins-bin/files/jenkins-bin-systemd.confd

```diff
@@ -0,0 +1,68 @@
+# Config file for /etc/init.d/jenkins
```

Comment:
This file is pretty much similar to jenkins-bin.confd, and the added JAVA_COMMAND_LINE won't hurt in the openrc case so it would seem better and more maintainable to use the same file?

Reply:
@graaff I will change this out of aesthetic reason, it was taken directly from the confd one 👍